Fix `config.py` tool according to the new API of fido2 python package

jmichelp opened the pull request:
Fixes #269

Since fido2 v0.8.1 which we developed our configuration tool on, the device descriptor
moved from python `dict` to `NamedTuple`. This PR updates to this new API and ensure 
that we're using the correct version of fido2 module.

- [x] Tests pass
- [x] Appropriate changes to README are included in PR

jmichelp committed:
Fix fido2 API update.

Since fido2 0.8.1 the device descriptor moved to NamedTuple, breaking
our configuration tool.
Code is now updated accordingly and the setup script ensure we're
using the correct version for fido2 package.

jmichelp commented:
Considering we're not using virtualenv here, I'm unsure whether we should use `fido2>=0.9.1` or `fido2==0.9.1` as a version description.
The latter may prevent people from updating the package or even cause an error (I'm not sure how pip behaves if it has to downgrade a package)

jmichelp committed:
Make Yapf happy

kaczmarczyck commented on setup.sh:
```diff
@@ -46,4 +46,4 @@ mkdir -p elf2tab
 
 # Install python dependencies to factory configure OpenSK (crypto, JTAG lockdown)
-pip3 install --user --upgrade colorama tqdm cryptography fido2
+pip3 install --user --upgrade colorama tqdm cryptography "fido2>=0.9.1"
```
If this list is growing, it might make sense to go for
```shell
pip3 install -r requirements.txt
```

and have the packets in a `requirements.txt` doc? We also have i.e. `tools/configure.py`, and the requirements could then indicate more easily which requirement is from where when things get more complicated.

jmichelp replied:
That's something we can do in the `develop` branch for the next release. For `stable` I would like to keep it as a small bugfix.

kaczmarczyck replied:
Sounds good!

jmichelp committed:
Fix missing update for fido2 0.9.1

Also split the comment into 2 lines so that the touch is not hidden
at the end of the screen.
